src/globals/scss/helpers/_spacing.scss

```diff
@@ -0,0 +1,39 @@
+// Create responsive spacing as set in settings/spacing.
+
+// Create responsive margins
+// Arguments:
+// $value = top \ right \ bottom \ left \ all
+@mixin govuk-spacing-mq-margin($scale-map, $value: "all", $important: false) {
```

Comment:
Do we think we need both `spacing` and `margin` in the mixin name? Wonder if `govuk-margin-mq` would be clear enough?

Reply:
Fixed! See my comment below.